guides/source/routing.md

```diff
@@ -76,7 +91,7 @@ Rails.application.routes.draw do
 end
 ```
 
-Since this is a regular Ruby source file you can use all of its features to help you define your routes but be careful with variable names as they can clash with the DSL methods of the router.
+Since this is a regular Ruby source file you can use all of its features to help you define your routes. (Be careful with variable names as they can clash with the DSL methods of the router.)
```

Comment:
Feel free to ignore this as it's more of a personal preference. 

I preferred the initial versions here because it connects the two statements. I also find it slightly unusual (while grammatically acceptable) to have a fully independent sentence in parenthesis.

```suggestion
Since this is a regular Ruby source file you can use all of Ruby's features to help you define your routes but be careful with variable names as they can clash with the DSL methods of the router.
```